user-service/pom.xml

```diff
@@ -25,11 +25,15 @@
         <relativePath>../pom.xml</relativePath>
     </parent>
 
+    <!--  *********** Aritfact ID ***********    -->
     <artifactId>user-service</artifactId>
+    <url>https://github.com/gchq/Palisade-services/tree/develop/user-service</url>
+    <!--  *********** Aritfact ID ***********    -->
 
     <properties>
         <spring-cloud-starter-ribbon.version>1.4.7.RELEASE</spring-cloud-starter-ribbon.version>
         <swagger.version>1.5.21</swagger.version>
+        <eureka.version>1.9.16</eureka.version>
```

Comment:
No point adding this as it's getting removed in PAL-653

Reply:
Will remove this.